### DISCUSSION TITLE
Popup placement

### DISCUSSION THREAD

**Discussion by tgb1964** [Clooos/Bubble-Card#596] [Feature requests]
I am using bubble card for popups on a tablet that is divided into 3 columns.  I'd like to be able to have it so the popups cover one of the three columns.  As it is, it will only popup into the center.  I've tried putting the popup card into a layout card but it didn't work.  This would be a nice feature to have (unless it is already a feature and I didn't find it).

**Comment by Clooos** [comment]
Hi, you can add this custom style to change its position, just tweak the `100px` value, I hope this help.

```yaml
styles: |
  @media only screen and (min-width: 600px) {
    .bubble-pop-up {
      left: 100px !important;
    }
  }
```

**Reply by tgb1964** [reply]
Outstanding!  Thanks for the quick help.  Now...
any way to remove the X (close popup) icon from the top?  
I'm guessing it is something similar but I don't know what it would be called.

**Reply by Clooos** [reply]
The best way to fully remove the header is probably to do that, it's not a simple custom style but I'm planning to add the possibility to fully remove it with a setting in the editor, it will be easier.

```yaml
.bubble-header-container {
  display: none !important;
}
.bubble-pop-up-container {
  padding: 0 !important;
  margin-top: -14px !important;
  mask-image: none !important;
  -webkit-mask-image: none !important;
}
```

**Reply by tgb1964** [reply]
Again--works like a charm.  Thanks. Alas, I may need the header after-all.  I want to be able to independently control popups for all 3 columns so would need the X to shutdown that popup specifically. A smaller X would be nice though.

**Reply by Clooos** [reply]
You can't open multiple pop-ups at the same time and there is nothing to change that, but I'm not sure that this is what you want. You can still click outside of the pop-up to close it or by sliding down inside of it if you want.

**Reply by Clooos** [reply]
But or the close button you can change it like this, I also removed the background in this example.

```yaml
.bubble-close-button {
  --mdc-icon-size: 20px;
  background: none !important;
}
```

You can take a look at the documentation in the Styling section, I've added many examples and how to find the styles you can edit.

**Reply by tgb1964** [reply]
No multiple?  Even is separate vertical stacks?  That is what I was headed towards trying.  Thanks again.

**Reply by Clooos** [reply]
Yes you can have multiple pop-ups of course, but you can only have one opened at a time. Opening another one will close the previous one.
